gatsby-config.js

```diff
@@ -22,6 +22,14 @@ const ignoreFolders = process.env.BUILD_FOLDERS
       )
       .map((folder) => `${__dirname}/src/content/docs/${folder}/*`)
   : [];
+const i18nIgnoreFolders = additionalLocales
+  .map((locale) => {
+    if (process.env.LOCALE === locale) {
+      return null;
+    }
+    return `${__dirname}/src/i18n/content/${locale}/*`;
+  })
+  .filter(Boolean);
```

Comment:
if you flip these you can avoid the conditional in the `map` 😃 

```suggestion
const i18nIgnoreFolders = additionalLocales
  .filter((locale) => process.env.LOCALE !== locale)
  .map((locale) => `${__dirname}/src/i18n/content/${locale}/*`);
```

Reply:
😂 ahhh wonderful!